docs/helping.md

```diff
@@ -0,0 +1,85 @@
+# Helping
+
+This section is meant mostly for those that will be actively helping others in the Northstar Discord server or otherwise.
+If you've somehow stumbled upon this page trying to fix an issue for yourself, you should try checking the [troubleshooting page](installing-northstar/troubleshooting.md) instead. 
+If you're a helper, it also isn't a bad idea to look at this page a few times and try to have a rough idea of the things that are on it.
+
+When helping a user, please try to move them to the `#help` channel to create a ticket as well as to keep public channels less cluttered. 
+
+If they really don't want to open a ticket or you don't want to deal with a ticket, ask them to move to `#northstar-chat`. 
+There, not only can they embed things, but it'll be slightly more focused on helping than other channels would be.
+
+Arguably the most important things to know as a helper are the ability to read logs and deal with users that don't read the wiki (90% of tickets are solutions solvable by checking the wiki.
+There is a pretty high chance that even if the user claims they read the wiki they didn't actually do so or didn't find their issue on there)
+
+Reading logs is a little tougher however, but once you get the basics down you can easily diagnose issues in a few seconds after opening a log. 
+You can use `/tag id:logs` to inform users how to properly send a log in a Discord channel. 
+Generally, when reading a log there are a list of things you want to look for, which will be covered later.
+
+The first section will first go over extremely common issues and their tags, then the basics of using the ticket bot for commands inside tickets, and then some overview and specific examples for reading logs.
+
+Additionally, please don't be afraid to ask for help. The entirety of `#volunteers-chat` in the Discord server is specifically for those that help/sometimes help users and there's no shame it not knowing.
+
+## Common issues and their tags
+
+In the Northstar Discord server, a massively used device to help end users is the tickets bot. A lot of the common issues and questions that come up have tags for them on the Discord server. 
+The tag command for the tickets bot is `/tag`, and you can either type `/tag id:{TAG ID HERE}` or let the bot autofill the tag for you after typing `/tag` and selecting the tag id from the list (make sure you select the ticket bot, and not Dyno!). 
+Note, you can't reply to a user while using a slash command, so try to reply to the bot's message after sending the tag command while pinging the person it's directed at and informing them that they should follow the provided solution.
+
+You can find a list of these tags on the [tag dashboard for the help bot](https://panel.ticketsbot.net/manage/920776187884732556/tags), so long as you sign in using your Discord account and have access to it with the roles that you have.
```

Comment:
This link causes linkcheck to fail due to missing access. The easiest solution is to add it to its ignore list under `.github/.markdownlinkcheck.json`